munch-match/src/components/ShareModal/ShareModalContent.jsx

```diff
@@ -4,52 +4,65 @@ import ShareButton from "./ShareButton";
 
 const ShareModalContent = ({ imageUrl, name, address, shareUrl }) => {
     const handleCopyLink = () => {
+        const linkField = document.querySelector(`.${styles.linkField}`);
```

Comment:
Don't need this can just write the `shareUrl` prop to clipboard

Reply:
It copies "undefined" without that assignment

Reply:
Yes ma'am have fixy now